src/Symfony/Bridge/Twig/Resources/views/Form/tailwind_2_layout.html.twig

```diff
@@ -0,0 +1,82 @@
+{% extends 'form_div_layout.html.twig' %}
```

Comment:
should use `{% use %}` instead, so that this template stays usable for `{% use %}` tags too